shapely/geometry/base.py

```diff
@@ -721,23 +721,22 @@ def dwithin(self, other, distance):
         """
         return _maybe_unpack(shapely.dwithin(self, other, distance))
 
-    def equals_exact(self, other, tolerance):
-        """True if geometries are equal to within a specified
-        tolerance.
+    def equals_exact(self, other, tolerance=0.0, normalize=False):
+        """Returns True if the objects have the same sequence of coordinates
+        within a certain tolerance (unlike `self.equals`, which tests
+        topological equality).
+
+        Refer to `shapely.equals_exact` for full documentation.
 
         Parameters
         ----------
         other : BaseGeometry
             The other geometry object in this comparison.
-        tolerance : float
+        tolerance : float, optional (default: 0.)
             Absolute tolerance in the same units as coordinates.
-
-        This method considers coordinate equality, which requires
-        coordinates to be equal and in the same order for all components
-        of a geometry.
-
-        Because of this it is possible for "equals()" to be True for two
-        geometries and "equals_exact()" to be False.
+        normalize : bool, optional (default: False)
```

Comment:
should be `normalize : bool default False`. optional means that it accepts `None` as an input, i.e. `Optional[bool]`.

Reply:
Related: https://github.com/shapely/shapely/issues/1701

Reply:
> > optional means that it accepts None as an input, i.e. Optional[bool].
> 
> I'm not sure this is true? According to [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html#parameters), `optional` in the docstring like this means that there are default values, which can be of the same type.

I see your reference in numpydoc, but I find it very confusing. Why would you use the keyword `optional` which has a specific meaning in python when you actually mean `default`.  I also can't find such use of the keyword `optional` in shapely docs.

Reply:
the use of ``optional`` in docstrings predates annotations and historically was used to indicate that the argument is optional in the sense that you do not need to provide it because it has a default value; this is the meaning that users understand by default; only developers may link it to the ``Optional`` annotation.

Reply:
`optional` is used in many places in shapely docstrings; e.g., [points constructor](https://shapely.readthedocs.io/en/stable/reference/shapely.points.html#shapely.points) for exactly the reasons provided by @tfardet ; at least some of these came via PyGEOS.  `default: <value>` is rarely used in the docstrings; I believe when this came up in PyGEOS the preference there was that the default `kwarg` parameter values was sufficient documentation because that was easily visible in the function definition shown in the docs (i.e., using `default: <value>` was viewed as redundant).